open-reac/src/main/java/com/powsybl/openreac/parameters/OpenReacAmplIOFiles.java

```diff
@@ -54,6 +56,7 @@ public OpenReacAmplIOFiles(OpenReacParameters params, Network network, boolean d
         //outputs
         this.reactiveSlackOutput = new ReactiveSlackOutput();
         this.networkModifications = new NetworkModifications(network);
+        this.voltagePlanOutput = new VoltagePlanOutput();
```

Comment:
In general, we use `profile` and not plan. See short-circuit analysis API.

Reply:
Indeed, thank you.